### PR TITLE
Revert "Issue 1"

### PR DESCRIPTION
**Pull request by deeheber**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

It appears this repo was initialized with issue 1 completed. Thanks for pointing this out @ahsanatzapier!

## Related Issue

1

## Acceptance Criteria

As mentors, we need to be able to demo pair programming at our first sync using issue 1 as an example. In order to do so, the code to complete issue 1 should not be present.

## Type of Changes

Removing the code to complete issue 1.

### Before

Before the code to complete issue 1 (show sign in and sign out buttons) was there.

### After

After the code to complete issue 1 is gone. the sign in and sign out buttons should not be visible or present on the page.

## Testing Steps / QA Criteria

1. I made the code changes
2. Ran `npm start` 
3. Viewed the site at `http://localhost:3000/`
4. Verified I could no longer see the sign in and sign out buttons

### CODE REVIEW

File: src/views/Layout.jsx [hunk 1]
```diff
@@ -1,8 +1,7 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```
Comment (deeheber) [inline src/views/Layout.jsx:1]: This was needed bc the git hooks did not let me commit.

Once the issue 3 is complete we can guide the collabies to delete this.

Comment (ahsanatzapier) [reply src/views/Layout.jsx:1]: Thanks for adding this!

Comment (ahsanatzapier) [reply src/views/Layout.jsx:1]: Revert changes look great! 🚀

**Review by ahsanatzapier — APPROVED** [review]
Revert changes look great! 🚀

**Pull request merged**